definitions/npm/indent-string_v3.2.x/flow_v0.25.x-/indent-string_v3.2.x.js

```diff
@@ -0,0 +1,10 @@
+declare module 'indent-string' {
+  declare module.exports: (
+    input: string,
+    count?: number,
+    opts?: {
+      includeEmptyLines?: bool,
+      indent?: string,
+    }
```

Comment:
Could you please make that object exact and its keys covariant?
```
opts?: {|
  +includeEmptyLines?: bool,
  +indent?: string,
|}
```

See 
https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#prefer-immutability  
https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#prefer-exactness

Reply:
I remember seeing that in the guide but forgot for my PR. Thanks! It's applied now.